fastlane/Fastfile

```diff
@@ -59,6 +59,94 @@ lane :github_release do |options|
   )
 end
 
+desc "Update dependencies to latest GitHub releases"
+lane :update_native_dependencies_to_latest do |options|
+  latest_ios_release_info = get_latest_github_release(url: "RevenueCat/purchases-ios")
+  latest_android_release_info = get_latest_github_release(url: "RevenueCat/purchases-android")
+
+  latest_ios_release_name = latest_ios_release_info["name"]
+  latest_android_release_name = latest_android_release_info["name"]
+
+  UI.message("ℹ️  Latest RevenueCat iOS version: #{latest_ios_release_name}")
+  UI.message("ℹ️  Latest RevenueCat Android version: #{latest_android_release_name}")
+
+  update_ios_native_versions(version: latest_ios_release_name)
+  update_android_native_versions(version: latest_android_release_name)
+
+  run_pod_install
+end
+
+desc "Update purchases-ios dependency version"
+lane :update_ios_native_versions do |options|
```

Comment:
Hmm I feel this should be singular? As in `update_ios_native_version`. Same for the android lane.